parsing/lexer.mll

```diff
@@ -1045,7 +1022,7 @@ and skip_hash_bang = parse
   let init () =
     begin match Option.map parse_keyword_edition !Clflags.keyword_edition with
     | None -> ()
-    | Some (edition,(remove,add)) -> set_keyword_edition ~remove ~add edition
+    | Some (edition, add) -> set_keyword_edition ~add edition
```

Comment:
Note: if we are to review commit-by-commit, I would encourage you to squash both commits so that I only see the simpler version. (You can extract the support for keyword removal as a separate patch by just using `git revert` to this one, and then keep it around in a separate branch in case the people ask for it again.)